README.md

```diff
@@ -4,7 +4,12 @@ TenSEAL is a library for doing homomorphic encryption operations on tensors, bui
 
 ## Installation
 
-TODO
+Install CMake. 
+If you are on Linux, head down to [CMake- Snap](https://snapcraft.io/cmake) and install CMake.
+
+Run ```prereqs``` to setup dependecies.
```

Comment:
I think we should explain what the user should do here (get those third party library) instead of providing a bash script that's somehow platform dependent.

Reply:
What can be one approach is use Docker for development, so development is only done on Linux. So we don't need to worry about supporting anything else. 

But surely, I'll add the instructions just in case for non Linux development.

Reply:
Done!

Reply:
I'm working out a Dockerfile which should be available today.